### ISSUE TITLE
ignore ~~CLOSETOC~~ in included pages

### ISSUE THREAD

**Issue by ssahara** [ssahara/dw-plugin-toctweak#5]
The `~~CLOSETOC~~` macro should be specified in the current page.

**Comment by ssahara** [comment]
fixed in commit 654fdbbf8443378e949d782e8b53361ad945d57e

**Comment by ralfjunker** [comment]
I don't agree that the `~~CLOSETOC~~` macro should be specified in the current page. Please see issue #9 for details.

**Comment by ssahara** [comment]
The initial state of built-in TOC (none, opened or closed) should be controllable per page basis, as same as `~~NOTOC~~`.  Therefore, this falls into idea that `~~CLOSETOC~~` macro should be specified in the current page.

Aside from above, there are no config parameter to set default state of visible TOC, i.e. closed or opened. 
It may be useful to have such config in this plugin instead of including common page or section that contains `~~CLOSETOC~~`?
